### PR TITLE
refactor(testing): improve test readability and consistency

### PR DESCRIPTION
**Pull request by appleboy**
- Refactor test code in `client_test.go` to use a local variable `item` instead of `tt` within test cases for `CanPerformStartTLS` and `CanPerformTLS` functions.

* [X] Tested and/or added tests to validate that the changes work as intended, if applicable.
* [X] Updated documentation in README.md, if applicable.

### CODE REVIEW

**Comment by appleboy** [comment]
@TwiN Please help to review.

File: client/client_test.go [hunk 1]
```diff
@@ -122,15 +122,16 @@ func TestCanPerformStartTLS(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
```
Comment (TwiN) [inline client/client_test.go:124]: If you're looking for consistency, I'd suggest changing 
```
for _, tt := range tests {
```
for 
```
for _, scenario := range scenarios {
```
as it's what I name test cases in most Gatus tests

Comment (appleboy) [reply client/client_test.go]: @TwiN Done [`0ae4b63` (#664)](https://github.com/TwiN/gatus/pull/664/commits/0ae4b63c4212f47cdb758077be479e8a7a139788)

File: client/client_test.go [hunk 2]
```diff
@@ -147,16 +147,17 @@ func TestCanPerformStartTLS(t *testing.T) {
 			wantErr:       false,
 		},
 	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
+	for _, scenario := range scenarios {
+		item := scenario
```
Comment (TwiN) [inline client/client_test.go:151]: I don't really understand what this is changing here: the `item` variable isn't necessary, you can just use `scenario` directly 🤔

File: client/client_test.go [hunk 3]
```diff
@@ -199,15 +200,16 @@ func TestCanPerformTLS(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		item := tt
```
Comment (TwiN) [inline client/client_test.go:203]: Same comment as above - for consistency, I'd rename `tests` to scenarios`, get rid of `item := tt` and rename `tt` to `scenario`

**Review by TwiN — CHANGES_REQUESTED** [review]

**Comment by TwiN** [comment]
Closing this, as I don't really see what improvement this brings.

Regardless, thank you for contributing!

**Pull request closed without merging**